boards/FSM/Inc/main.h

```diff
@@ -71,14 +71,9 @@ extern "C"
 #define TASK1KHZ_STACK_SIZE 128
 #define TASKCANRX_STACK_SIZE 128
 #define TASKCANTX_STACK_SIZE 128
-#define TIM2_FREQUENCY 72000000
-#define TIM2_AUTO_RELOAD_REG 0xFFFF
-#define TIM2_PWM_MINIMUM_FREQUENCY 1
-#define TIM2_PRESCALER \
-    (TIM2_FREQUENCY / TIM2_AUTO_RELOAD_REG / TIM2_PWM_MINIMUM_FREQUENCY)
-#define TIMx_AUTO_RELOAD_REG 0xFFFF
+#define TIM4_AUTO_RELOAD_REG 0xFFFF
 #define TIMx_FREQUENCY 72000000
-#define TIMx_PRESCALER (TIMx_FREQUENCY / TIMx_AUTO_RELOAD_REG)
+#define TIM4_PRESCALER (TIMx_FREQUENCY / TIM4_AUTO_RELOAD_REG)
```

Comment:
@pasitlaothamatas Why did you get rid of `PWM_MINIMUM_FREQUENCY`. I think it would be helpful to remind the reader what range of PWM frequencies TIM4 can measure. And it wouldn't seem like the auto reload register value was arbitrarily chosen.